doc/getting-started/how-to-install-smash.md

```diff
@@ -72,20 +83,26 @@ The most important points of it are here:
 
 70+50+255+5+64 = 444 bytes.
 
-## How to run it with the node
+## How to run SMASH with the Cardano node
+
+First run the Cardano node as a relay, e.g.:
 
-First run the node. The genesis should be generated from Nix. You can run smash with:
 ```
-SMASHPGPASSFILE=./config/pgpass stack run smash-exe -- run-app-with-db-sync --config ../cardano-db-sync/config/testnet-config.yaml --genesis-file /nix/store/pdnsmv1gi6llr92fk7sgs3j2gcphm6fm-shelley-testnet-genesis.json --socket-path ../cardano-node/state-node-shelley_testnet/node.socket --schema-dir schema/
+cardano-node --genesis-file genesis.json --socket-path node.socket --config config.yaml
+```
+
+You can then run ``smash`` using e.g:
+```
+SMASHPGPASSFILE=./config/pgpass stack run smash-exe -- run-app-with-db-sync --config config.yaml --genesis-file genesis.json --socket-path node.socket --schema-dir schema/
```

Comment:
We no longer need the `genesis-file` path, it's now included in the `config.yaml`.